packages/blade/src/components/VisuallyHidden/ScreenReaderStyles.ts

```diff
@@ -10,6 +10,7 @@ const screenReaderStyles: CSSObject = {
   padding: 0,
   position: 'absolute',
   width: `1px`,
+  left: '-10000px',
```

Comment:
Seems like a hacky fix, is this the only way 👀

Reply:
Perhaps add a comment in the code for future reference

Reply:
Yeah definitely "looks" hacky, will add the link to https://webaim.org/techniques/css/invisiblecontent/#techniques which explains the properties being used.

Reply:
If there is an ultra wide screen display, would this end up being visible to the user?

Reply:
> end up being visible

No it will still be invisible, the other properties will ensure that. 

Though it's quite unlikely, zooming out to 25% on my mac only goes to 5760px. 😅 the designs will most likely break first at those extreme screen sizes.